Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+    process_command(["cmake", "--build", non_mono_build_path, "--target", f"{project_name}.Assets", "--config", "profile", "--", "/m"])
+
+def bundle_assets(project_path: pathlib.Path,
+                  selected_platform: str,
+                  seedlist_paths: List[str],
+                  non_mono_build_path: pathlib.Path):
+    asset_bundler_batch_path = non_mono_build_path / 'bin' / 'profile' / ('AssetBundlerBatch' + ('.exe' if platform.system().lower()=='windows' else ''))
+    asset_list_path = project_path / 'AssetBundling' / 'AssetLists'
+
+    game_asset_list_path = asset_list_path /  f'game_{selected_platform}.assetlist'
+    engine_asset_list_path = asset_list_path / f'engine_{selected_platform}.assetlist'
+    bundles_path = project_path / 'AssetBundling' / 'Bundles'
+
+    #generate the asset lists for the project
+    gen_game_asset_list_command = \
+        f"{asset_bundler_batch_path} assetLists --assetListFile {game_asset_list_path} --platform {selected_platform} --allowOverwrites"
+    for seed in seedlist_paths:
+        gen_game_asset_list_command += " --seedListFile " + str(seed)
+    process_command(gen_game_asset_list_command.split())
+
+    gen_engine_asset_list_command = \
+        f"{asset_bundler_batch_path} assetLists --assetListFile {engine_asset_list_path} --platform {selected_platform} --allowOverwrites --addDefaultSeedListFiles"
+    process_command(gen_engine_asset_list_command)
+
+    #generate the bundles. We will place it in the project directory for now, since the files need to be copied multiple times (one for each separate launcher distribution)
+    process_command([asset_bundler_batch_path, "bundles", "--maxSize", "2048", "--platform", selected_platform, "--allowOverwrites",
+                         "--outputBundlePath", bundles_path / f"game_{selected_platform}.pak",
+                         "--assetListFile", game_asset_list_path])
+    
+    process_command([asset_bundler_batch_path, "bundles", "--maxSize", "2048", "--platform", selected_platform, "--allowOverwrites",
+                         "--outputBundlePath", bundles_path / f"engine_{selected_platform}.pak",
+                         "--assetListFile", engine_asset_list_path])
+    return bundles_path
+
+
+def create_launcher_layout_directory(project_path: pathlib.Path,
+                                     output_path: pathlib.Path,
+                                     platform: str,
+                                     mono_build_path: pathlib.Path,
+                                     bundles_to_copy: List[pathlib.Path],
+                                     gem_folder_names_to_copy: List[str],
+                                     project_file_patterns_to_copy: List[str],
+                                     build_config: str,
+                                     archive_output_format: str = "none",
+                                     logger: logging.Logger|None =None):
```

Comment:
nitpick: missing space between = and None
```suggestion
                                     logger: logging.Logger|None = None):
```